src/engraving/libmscore/cmd.cpp

```diff
@@ -1909,6 +1911,21 @@ void Score::toggleAccidental(AccidentalType at, const EditData& ed)
 
 void Score::changeAccidental(AccidentalType idx)
 {
+    if (selection().isAccentidentalsOnly()) {
```

Comment:
I think we can remove this "if" condition and return statement; we should allow a mixed selection. For example, if my selection is something like in the first measure of the following example, I would like to end up with something like the second measure:
<img width="273" alt="Scherm­afbeelding 2022-12-02 om 23 26 46" src="https://user-images.githubusercontent.com/48658420/205400462-844bb5ea-3534-4488-a69f-4e191ce3dbee.png">

Reply:
Do I then remove the `isAccentidentalsOnly()` method (as it's not used anymore)?

Reply:
Yes, indeed.